### ISSUE TITLE
upsample output size missing?

### ISSUE THREAD

**Issue by blueardour** [onnxsim/onnxsim#8]
Hi, @daquexian  Thanks for the greate project. It helps to solve a problem in my recent work.

However, I noticed one function was cancelled in a more recent commit. I was updating a model with upsample layer. Its output size was computed and recorded in the graph in your last commit. Today I sychnoized your code, and found the output size was on longer computed. 
![Screenshot from 2019-05-31 15-57-45](https://user-images.githubusercontent.com/4530223/58686065-db6db080-83bc-11e9-8732-1804fac905d2.png)

Was there any reason for not computing the output size  dynamically?

**Comment by daquexian** [comment]
Thanks for your report!

The shape inference feature is provided by onnx instead of onnx-simplifier, so maybe an upgrade of onnx breaks it on your model.

However, if you want you can send your model to me by email (my email is daquexian566@gmail.com). I'm glad to help you check what the reason is.

**Comment by daquexian** [comment]
Thanks for your model!

I have checked it, however, I cannot reproduce it on my side.

Could you please provide your onnx and onnx-simplifier version? Mine is:

```python
>>> import onnx
>>> print(onnx.__version__)
1.5.0
>>> import onnxsim
>>> print(onnxsim.__version__)
0.1.5
```

**Comment by daquexian** [comment]
About the "fake upsample" in your email:

> I got a network with some 'fake upsample,' by which the input feature map and output feature map have the same shape. I noticed you could deleted some constant node and fuse BN into conv. I wonder would it be possilble to check the upsample layers (once found a upsample with the same input/output shape, we should elimate it)?

It is easy to implement by adding an [optimizer](https://github.com/onnx/onnx/blob/master/docs/Optimizer.md) in ONNX. However, if we implement it in this way, we cannot use it in onnx-simplifier until the next release of ONNX is published, and you need to build ONNX and [run the optimizer](https://github.com/onnx/onnx/blob/master/docs/PythonAPIOverview.md#optimizing-an-onnx-model) by yourself.

**Comment by blueardour** [comment]
Hi, @daquexian  You are so kindful. Thanks very much for helping debug the issue.

Today, I made some test.  My onnx version is the same with you (1.5.0). And I traced back of the onnx-simplier version when the umsample  output shape was correctly given out. 

From my result, it seemed for onnxsim of version 1.3.0, the output shape could be computed.
![Screenshot from 2019-06-04 17-14-11](https://user-images.githubusercontent.com/4530223/58860860-3b7d9300-86ec-11e9-9dd4-58bc474e724e.png)

And the output shape was missing after that version, for example, the 1.4.0 and 1.5.0 gave:
![Screenshot from 2019-06-04 17-16-19](https://user-images.githubusercontent.com/4530223/58860966-80092e80-86ec-11e9-9f1c-aa0e7ddc1b69.png)

**Comment by daquexian** [comment]
Sorry, but I'm afraid I have not got your point. It looks all shapes are
presented even in the latter image.

**Comment by daquexian** [comment]
Thanks for your reply. I'm afraid I have no good way to fix it since it is not reproducible on my side. Sorry for that.

**Comment by xylcbd** [comment]
I exported a model using onnx-simplifier, the code snip here:

`cx1 = torch.nn.functional.interpolate(cx1, size=(x.size(2)//8, x.size(3)//8), mode='bilinear', align_corners=False)`

![onnx](https://user-images.githubusercontent.com/3643991/58941033-88756e00-87ad-11e9-9745-e42945a021ed.png)

It seems that the output size of upsample is (0, 0), which is unexpected! Any suggestions? thx!

**Comment by blueardour** [comment]
@xylcbd  If the code is exported from pytorch, as far as I could remeber,  one should fix the height and width explictly in the code. It's an unsolved issue of the pytorch/onnx.

**Comment by blueardour** [comment]
@daquexian  Thanks for paying so many time on the issue. Currently, I could work around it by checkout into 1.3.0  version. I'd like to dig the issue when I get more familiar with this repo.

**Comment by lxgyChen** [comment]
> Thanks for your reply. I'm afraid I have no good way to fix it since it is not reproducible on my side. Sorry about that.
@blueardour I face the same problem, and here is my code:
`
import torch.nn as nn
import torch.nn.functional as F
import torch.onnx
import onnxsim

class Net(nn.Module):
    def __init__(self):
        super(Net, self).__init__()
        self.conv1_1 = nn.Conv2d(3, 16, 3, padding=1)
        self.conv1_2 = nn.Conv2d(16, 16, 3, padding=1)
        self.bn1 = nn.BatchNorm2d(16)
        self.maxpool = nn.MaxPool2d(2, stride=2)
        self.conv2_1 = nn.Conv2d(16, 32, 3, padding=1)
        self.conv2_2 = nn.Conv2d(32, 32, 3, padding=1)
        self.bn2 = nn.BatchNorm2d(32)
        self.conv3_1 = nn.Conv2d(32, 16, 3, padding=1)
        self.conv3_2 = nn.Conv2d(16, 16, 3, padding=1)
        self.conv3_3 = nn.Conv2d(16, 3, 3, padding=1)
        self.bn3 = nn.BatchNorm2d(3)

    def forward(self, x):
        x1 = F.relu(self.bn1(self.conv1_2(F.relu(self.conv1_1(x)))))
        x2 = self.maxpool(x1)
        xup = F.interpolate(x2, scale_factor=2, mode='bilinear', align_corners=False)
        x3 = self.bn2(self.conv2_2(F.relu(self.conv2_1(xup))))
        x4 = F.relu(self.conv3_3(self.conv3_2(F.relu(self.conv3_1(x3)))))
        x5 = self.bn3(x4)
        return F.softsign(x5)


inp = torch.rand(1, 3, 96, 96)
net = Net().cuda()
torch.onnx.export(net, inp.cuda(), "..//model//net.onnx",  export_params=True, 
                              opset_version=9)`
My PyTorch version is 1.2.0, onnx version is 1.5.0, onnxsim version is 0.1.5
